app/views/volunteers/_form.html.slim

```diff
@@ -1,3 +1,12 @@
+- if policy(Volunteer).destroy?
```

Comment:
This seems clunky, if the current_user can destroy the volunteer, than he should be able to change the following stuff. I whould rather use if `current_user.superadmin?` as it says which role can do the following actions